ludwig/features/image_feature.py

```diff
@@ -637,6 +806,34 @@ def __init__(self, input_feature_config: ImageInputFeatureConfig, encoder_obj=No
         else:
             self.encoder_obj = self.initialize_encoder(input_feature_config.encoder)
 
+        # set up for augmentation if specified
+        if input_feature_config.augmentation:
```

Comment:
So based on this comment and how you declared the `augmentation` parameter on the image feature schema you will explicitly allowing a user to set `augmentation` to `None` (though the default if they specify nothing will be `False`). Want to confirm if that is what you intended.

Reply:
@ksbrar 
Not the intent.   `None` should be flagged as an error.  The only valid values should be
* False
* True
* List of augmentation operations.

That code was written before the decision was made to allow boolean values. 

What is the best way to resolve this?  check explictly for `None` and raise an exception if it is `None`.

Reply:
@jimthompson5802 You don't need to do anything here. I would just change the comment to # set up for augmentation if its enabled. Validation will handle any unsupported parameters before this code path gets hit, and then when it finally does this if statement should logically only be checking if `augmentation` is `False` (and if its `True` or a `List` then as it already does it'll set up the pipeline).

I'll add the details for the fix to the other thread.

Reply:
got it.  revised comment.